### ISSUE TITLE
Add focus indicator to goal icons on indicator pages

### ISSUE THREAD

**Issue by sarah-beasley** [open-sdg/open-sdg#1850]
example url - http://uk-sdg-feature-branches.s3-website.eu-west-2.amazonaws.com/feature-2.1.0-beta3/3-3-3/

It was highlighted from DAC that the goal icon images on each indicator page do not have focus indicators so the link cannot be used by keyboard and the border is non-existent. These need to be the same as other goals icons like the home page links - yellow with black inner border on default contrast mode. Please can you add to all indicator pages.
